src/app/visual/country-field/country-field-visual.component.ts

```diff
@@ -52,4 +52,8 @@ export class CountryFieldVisualComponent implements OnInit {
       this.disableFields = true;
     }
   }
+
+  public countryChanged(country: any) {
+    console.log('Change occurred: ' + country);
```

Comment:
Very minor, but is it alright to log in test code?

Reply:
Good catch. Didn't mean to check in the visual test changes. I've stripped that back out. The actual testing is done in the unit tests. I was just using this for local testing.